examples/rust/components/http-hello-world/wadm.yaml

```diff
@@ -16,9 +15,9 @@ spec:
     - name: http-component
       type: component
       properties:
-        # To use the locally compiled code in this folder, use the line below instead after running `wash build`:
-        # image: file://./build/http_hello_world_s.wasm
-        image: ghcr.io/wasmcloud/components/http-hello-world-rust:0.1.0
+        image: file://./build/http_hello_world_s.wasm
+        # To use the a precompiled version of this component, use the line below instead:
+        # image: ghcr.io/wasmcloud/components/http-hello-world-rust:0.1.0
```

Comment:
was this intentional?

Reply:
This *was* intentional, as people might be surprised when local changes don't appear

Reply:
@lachieh are you OK with this? Was playground going to try and use these `wadm.yaml`s?

Reply:
I think we should consider duplicating the `wadm.yaml` rather than switching between the local and external references, that way we could point the various examples at the appropriate version (for example, the various Kubernetes examples/documentation should use the external reference)

Reply:
Yeah I'm not against that -- naming might be tricky but two different files would make things easier/avoid having to do this toggling.

Reply:
Yeah, we've discussed before and I think what Joonas suggested is what we landed on, it just takes someone to implement. In the meantime, I'd prefer the examples to work out-of-the-box, which means external references. I think we have example manifests with both local and external, and the console has its own examples/dependencies so I don't really mind which happens in this PR.